lua/tardis/sh_menuoptions.lua

```diff
@@ -29,26 +29,107 @@ if CLIENT then
+			function visual_gui_big_toggle:OnChange(val)
+				TARDIS:SetSetting("visual_gui_bigpopup", val)
+			end
+			panel:AddItem(visual_gui_big_toggle)
+
+			local DLabel3 = vgui.Create( "DLabel" )
+			DLabel3:SetText("Number of visual GUI button rows on the screen:")
+			panel:AddItem(DLabel3)
+			local visual_gui_screen_numrows = vgui.Create("DNumSlider")
+			visual_gui_screen_numrows:SetMinMax(2, 10)
+			visual_gui_screen_numrows:SetDecimals(0)
+			visual_gui_screen_numrows:SetValue(TARDIS:GetSetting("visual_gui_screen_numrows"))
+			function visual_gui_screen_numrows:OnValueChanged(val)
+				val = math.floor(val)
+				visual_gui_screen_numrows:SetValue(val)
+				TARDIS:SetSetting("visual_gui_screen_numrows", val)
+			end
+			panel:AddItem(visual_gui_screen_numrows)
+
+			local DLabel4 = vgui.Create( "DLabel" )
+			DLabel4:SetText("Number of visual GUI button rows in the popup:")
+			panel:AddItem(DLabel4)
+			local visual_gui_popup_numrows = vgui.Create("DNumSlider")
+			visual_gui_popup_numrows:SetMinMax(2, 10)
+			visual_gui_popup_numrows:SetDecimals(0)
+			visual_gui_popup_numrows:SetValue(TARDIS:GetSetting("visual_gui_popup_numrows"))
+			function visual_gui_popup_numrows:OnValueChanged(val)
+				val = math.floor(val)
+				visual_gui_popup_numrows:SetValue(val)
+				TARDIS:SetSetting("visual_gui_popup_numrows", val)
+			end
+			panel:AddItem(visual_gui_popup_numrows)
+
+			local DLabel4 = vgui.Create( "DLabel" )
+			DLabel4:SetText("TARDIS Visual GUI Theme:")
+			panel:AddItem(DLabel4)
+
+			local visual_gui_theme = vgui.Create("DComboBox")
+			visual_gui_theme:SetText("Visual GUI Theme")
+			local themes = {}
+			local theme_basefolder = TARDIS.visualgui_theme_basefolder
+			local files, folders = file.Find(theme_basefolder.."*", "GAME", "nameasc")
+			for k,folder in pairs(folders) do
+				if file.Exists(theme_basefolder..folder.."/default_on.png", "GAME")
+					and file.Exists(theme_basefolder..folder.."/default_off.png", "GAME")
+					and file.Exists(theme_basefolder..folder.."/background.png", "GAME")
+					and file.Exists(theme_basefolder..folder.."/theme.json", "GAME")
+					-- File theme.json is not required yet, but might be needed in the future
+					-- it's easier to make all packs support it now than after they are
+					-- incompatible
+				then
+					local theme = {}
+					theme.ID = visual_gui_theme:AddChoice(folder)
+					theme.name = folder
+					table.insert(themes, theme)
+				end
+			end
+			local selectedtheme=TARDIS:GetSetting("visual_gui_theme", "default")
+			for k,theme in pairs(themes) do
+				if selectedtheme == theme.name then
+					visual_gui_theme:ChooseOption(theme.ID)
+					visual_gui_theme:SetText(theme.name)
+				end
+			end
+			visual_gui_theme.OnSelect = function(panel, index, value)
+				TARDIS:SetSetting("visual_gui_theme", value)
+				LocalPlayer():ChatPrint("TARDIS visual GUI theme changed to "..value)
+			end
+			panel:AddItem(visual_gui_theme)
```

Comment:
From testing: There isn't any padding at the bottom of the spawnmenu options
![image](https://user-images.githubusercontent.com/2363642/108837830-bffffb80-75ca-11eb-8993-f3b6d4efa150.png)

Reply:
https://github.com/MattJeanes/TARDIS/issues/360#issuecomment-785309643